src/evaluationSummaryAggregator.ts

```diff
@@ -0,0 +1,119 @@
+// TODO: flush on beforeunload
+
+// TODO: should we retry the data chunk if a flush fails?
+
+// TODO: pause when offline?
+
+// TODO: flush when we receive a config update (or as a result of a context update...but that should trigger a config update anyway)
+
+// TODO: client option to disable telemetry?
+
+import { PeriodicSync } from "./periodicSync";
+import { Config, ConfigEvaluationMetadata } from "./config";
+import ConfigValue from "./configValue";
+import { type prefab } from "./prefab";
+
+type ConfigValueType = "int64" | "string" | "bool" | "double";
+
+type SelectedValue = {
+  [key in ConfigValueType]?: ConfigValue;
+};
+
+type ConfigEvaluationCounter = Omit<ConfigEvaluationMetadata, "type"> & {
+  selectedValue: SelectedValue;
+  count: number;
+};
+
+type ConfigEvaluationSummary = {
+  key: string;
+  type: string; // should this be restricted type enum?
```

Comment:
It isn't clear to me if this is a question to yourself or the reviewers 😅 

I don't know that we need it but I don't have all the context